public/sass/_angular.scss

```diff
@@ -2171,3 +2172,51 @@ $easing: cubic-bezier(0, 0, 0.265, 1);
 div.flot-text {
   color: $text-color !important;
 }
+
+.gf-form-dropdown-typeahead {
```

Comment:
Why can't this one be moved to GlobalStyles or vice versa?

Reply:
it can, but there's no need since no one is using it. might as well remove it completely 👍